README.md

```diff
@@ -60,55 +60,26 @@ For known Mono defects, check [our issue tracker](https://github.com/castleproje
+`FEATURE_SYSTEM_CONFIGURATION`      | :white_check_mark: | :no_entry_sign:
+`FEATURE_TEST_COM`                  | :white_check_mark: | :no_entry_sign:
+`FEATURE_TEST_PEVERIFY`             | :white_check_mark: | :no_entry_sign:
+---                                 |                    |
+`DOTNET45`                          | :white_check_mark: | :no_entry_sign:
 
 * `FEATURE_APPDOMAIN` - enables support for features that make use of an AppDomain in the host.
 * `FEATURE_ASSEMBLYBUILDER_SAVE` - enabled support for saving the dynamically generated proxy assembly.
-* `FEATURE_BINDINGLIST` - enables support features that make use of System.ComponentModel.BindingList.
-* `FEATURE_DICTIONARYADAPTER_XML` - enable DictionaryAdapter Xml features.
-* `FEATURE_CUSTOMMODIFIERS` - enables reading and emitting optional and required custom modifiers defined on parameters including return parameters. It seems like a defect in corefx not to expose these methods because they are still implemented.
 * `FEATURE_EVENTLOG` - provides a diagnostics logger using the Windows Event Log.
-* `FEATURE_GAC` - enables support for obtaining assemblies using an assembly long form name.
-* `FEATURE_GET_REFERENCED_ASSEMBLIES` - enables code that takes advantage of System.Reflection.Assembly.GetReferencedAssemblies().
-* `FEATURE_IDATAERRORINFO` - enables code that depends on System.ComponentModel.IDataErrorInfo.
-* `FEATURE_ISUPPORTINITIALIZE` - enables support for features that make use of System.ComponentModel.ISupportInitialize.
-* `FEATURE_LEGACY_REFLECTION_API` - provides a shim for .NET 4.0 that emulates the `TypeInfo` API available in .NET 4.5+ and .NET Core.
-* `FEATURE_LISTSORT` - enables support for features that make use of System.ComponentModel.ListSortDescription.
-* `FEATURE_NETCORE_REFLECTION_API` - provides shims to implement missing functionality in .NET Core that has no alternatives.
 * `FEATURE_REMOTING` - supports remoting on various types including inheriting from MarshalByRefObject.
 * `FEATURE_SECURITY_PERMISSIONS` - enables the use of CAS and Security[Critical|SafeCritical|Transparent].
 * `FEATURE_SERIALIZATION` - enables support for serialization of dynamic proxies and other types.
-* `FEATURE_SMTP` - provides the email sender abstraction and implementation.
 * `FEATURE_SYSTEM_CONFIGURATION` - enables features that use System.Configuration and the ConfigurationManager.
-* `FEATURE_TARGETEXCEPTION` - enabled catching a `TargetException`. `System.Reflection.TargetException` is implemented by .NET Core but not exposed by corefx.
 * `FEATURE_TEST_COM` - enables COM Interop tests.
-* `FEATURE_TEST_DATASET` - enables tests that involve `System.Data.DataSet`s.
 * `FEATURE_TEST_PEVERIFY` - enables verification of dynamic assemblies using PEVerify during tests. (Only defined on Windows builds since Windows is currently the only platform where PEVerify is available.)
-* `FEATURE_TEST_SERILOGINTEGRATION` - enables Serilog integration tests.
```

Comment:
After this is merged we can talk through these, more definitely need to go.

Reply:
Agreed. I'll open an issue for that.